apps/web/src/scenes/CommunityPage/CommunityPageTabs.tsx

```diff
@@ -29,12 +29,27 @@ export default function CommunityPageTabs({ onSelectTab, activeTab, communityRef
             total
           }
         }
+        owners(
+          first: $listOwnersFirst
+          after: $listOwnersAfter
+          onlyGalleryUsers: $onlyGalleryUsers
+        ) @connection(key: "CommunityPageView_owners") {
+          # Relay doesn't allow @connection w/o edges so we must query for it
+          # eslint-disable-next-line relay/unused-fields
+          edges {
+            __typename
+          }
```

Comment:
do we need this? don't we just need pageInfo and total

Reply:
Here's a screenshot of the error if we remove them..
<img width="706" alt="CleanShot 2023-08-25 at 03 19 37@2x" src="https://github.com/gallery-so/gallery/assets/26466468/4128de99-2cd6-4eeb-9ab7-84c414e891e6">

Relay doesn't allow @connection without edges!